src/Utils.ts

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
I believe standard convention for utility providing files is to name them `utils.ts` (all lower case)